src/platforms/react-native/troubleshooting.mdx

```diff
@@ -172,11 +171,38 @@ If you're using `React Navigation 5` and you're not receiving transactions anymo
 
 If you experience mismatched line numbers on [sentry.io](https://sentry.io) when using RAM Bundles, this is due to a [bug](https://github.com/facebook/metro/issues/399) on the Metro tooling.
 
-## React Native 0.69 and Higher
+## Setup SKD v4 with React Native 0.69 and Higher
 
-If you're using React Native v0.69 or higher, the auto-linking has changed and we're working on a fix. Follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2267).
+To install an older version of the `@sentry/react-native` you need to specify it in the install command. Similarly, you will need to use compatible `@sentry/wizard` as shown in the script below.
 
-As a workaround, for now, update Sentry's `react-native.config.js` file, located in `node_modules/@sentry/react-native/react-native.config.js`.
+```bash
+yarn add @sentry/react-native@4
+npx @sentry/wizard@1 -i reactNative -p ios android
+npx pod-install
+```
+
+Before building the app check that the `android/app/build.gradle` was patched with the following Gradle integration:
+
+```groovy
+apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
+```
+
+This should work on Android, however, on iOS, it will still fail in the `Bundle React Native code and images` build step phase and for that, you have to revert the patched `shellScript` in the `ios/$projectName.xcodeproj/project.pbxproj` file, where `$projectName` is likely your project's name.
+Search for the build step phase `Bundle React Native code and images` and patch the `shellScript`.
+
```

Comment:
```suggestion
While this should work on Android, it will still fail in the `Bundle React Native code and images` build step phase on iOS. To fix this, revert the patched `shellScript` in the `ios/$projectName.xcodeproj/project.pbxproj` file, (where `$projectName` is likely your project's name). Then search for the build step phase `Bundle React Native code and images` and patch the `shellScript`.

```